internal/watcher/instance/nginx_process_parser.go

```diff
@@ -148,6 +150,12 @@ func (npp *NginxProcessParser) getInfo(ctx context.Context, proc *nginxprocess.P
 	nginxInfo.ExePath = exePath
 	nginxInfo.ProcessID = proc.PID
 
+	if confPath != "" {
```

Comment:
if you add back in the line  to getNGINXConfPath to parseNginxVersionCommandOutput() you can then do something like the following I believe 
```suggestion
	if  confPath := getConfPathFromCommand(proc.Cmd); if path != "" {
	      nginxInfo.ConfPath = confPath
	 }
```